.github/workflows/operate-test-importer.yml

```diff
@@ -11,9 +10,21 @@ on:
     paths:
       - '.github/workflows/operate-*'
       - "operate/**"
+# Limit workflow to 1 concurrent run per ref (branch): new commit -> old runs are canceled to save costs
+# Exception for main branch: complete builds for every commit needed for confidenence
+concurrency:
+  cancel-in-progress: true
+  group: ${{ format('{0}-{1}', github.workflow, github.ref == 'refs/heads/main' && github.sha || github.ref) }}
 jobs:
   run-importer-tests:
+    name: Operate importer integration tests
     uses: ./.github/workflows/operate-run-tests.yml
     with:
-      command: ./mvnw -pl operate -am verify -T1C -P skipFrontendBuild,operateItImport -B -Dfailsafe.rerunFailingTestsCount=2
+      command: ./mvnw -pl operate/qa/integration-tests verify -P skipFrontendBuild,operateItImport -B -Dfailsafe.rerunFailingTestsCount=2
+    secrets: inherit
+  run-zeebe-integration-tests:
+    name: Operate zeebe integration tests
+    uses: ./.github/workflows/operate-run-tests.yml
+    with:
+      command: ./mvnw -pl operate/qa/integration-tests verify -P skipFrontendBuild,operateItStage2 -B -Dfailsafe.rerunFailingTestsCount=2
```

Comment:
I think this is the only place we use `operateItStage2` profile - can we rename it something more meaningful? Even `operateZeebeIt` would might be sufficient